### ISSUE TITLE
auto repositioning bug fix

### ISSUE THREAD

**Issue by dcheslow** [Flyer53/jsPanel4#126]
jsPanel.js v 4.10.1
line 1483
old:
`var box = panel.options.container === 'window' ? document.body : panel.options.container;`
new:
`var box = panel.options.container === 'window' ? document.body : document.querySelector(panel.options.container);`

**Comment by Flyer53** [comment]
Hi there,
Bug is confirmed. Thanks for the report 👍 
But your proposed fix is not doing it because then it wouldn't work if option `container` is set already with a node object.
If you need the fix right now replace the line with
```javascript
let parent = panel.options.container;
const box = parent === 'window'
    ? document.body
    : typeof parent === 'string'
        ? document.querySelector(parent)
        : parent;
```
I'll have it fixed with the next release.

**Comment by Flyer53** [comment]
Fixed with v4.10.2 release
